pkg/conn/conn.go

```diff
@@ -289,6 +292,10 @@ func (mgr *Mgr) GetBackupClient(ctx context.Context, storeID uint64) (backup.Bac
 
 // ResetBackupClient reset the connection for backup client.
 func (mgr *Mgr) ResetBackupClient(ctx context.Context, storeID uint64) (backup.BackupClient, error) {
+	if ctx.Err() != nil {
+		return nil, errors.Trace(ctx.Err())
```

Comment:
if we check ctx.Err() here, I think the better way is remove `codes.Canceled` in RetryableError.

Reply:
OK, let's remove codes.Canceled from retryable list.